# TODO(mahdi): it could be an metaflow Exception class
class Fault(Exception):
    pass
